src/main/java/ua/hillel/moneymanager/controller/UserController.java

```diff
@@ -0,0 +1,31 @@
+package ua.hillel.moneymanager.controller;
+
+import org.springframework.boot.autoconfigure.neo4j.Neo4jProperties;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.core.Authentication;
+import org.springframework.web.bind.annotation.*;
+import ua.hillel.moneymanager.dto.UserDto;
+import ua.hillel.moneymanager.repository.UserRepository;
+import ua.hillel.moneymanager.service.user.UserService;
+
+@RestController
+@RequestMapping
```

Comment:
You must explicitly set a path to this controller. It can be `/api/auth`, for instance.